frontend/src/views/components/WizardForm/Page/Page.component.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import Boolean from 'views/components/WizardForm/FieldTypes/Boolean';
+import Single from 'views/components/WizardForm/FieldTypes/Single';
+import Multiple from 'views/components/WizardForm/FieldTypes/Multiple';
+import FreeAnswer from 'views/components/WizardForm/FieldTypes/FreeAnswer';
+import Rating from 'views/components/WizardForm/FieldTypes/Rating';
+
+import { FeedbackFieldTypes } from 'types/wizard-form.d';
+
+const Page: React.FC<{
+  title: string;
+  onPrevious: () => void;
+  handleSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+  questions: {
+    id: string;
+    type: FeedbackFieldTypes;
+    required?: boolean;
+    question: string;
+    description?: string;
+    answers: string[];
+    customAnswer?: {
+      id: string;
+      label: string;
+    };
+  }[];
+  previousButton?: string;
+  nextButton?: string;
+  submitButton: string;
+  isLastPage: boolean;
+  isFirstPage: boolean;
+  formValues: object;
+}> = (props) => {
+  const {
+    title,
+    questions,
+    onPrevious,
+    handleSubmit,
+    isFirstPage,
+    isLastPage,
+    previousButton,
+    nextButton,
+    submitButton,
+    formValues,
+  } = props;
+
+  const getAnswerComponent = (type) => {
+    switch (type) {
+      case FeedbackFieldTypes.Boolean:
+        return Boolean;
+      case FeedbackFieldTypes.Single:
+        return Single;
+      case FeedbackFieldTypes.Multiple:
+        return Multiple;
+      case FeedbackFieldTypes.FreeAnswer:
+        return FreeAnswer;
+      case FeedbackFieldTypes.Rating:
+        return Rating;
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit}>
+        <div className="m-wizard-form__form-content">
+          {!isFirstPage && title && (
+            <div className="m-wizard-form__form-content-title">
+              <h3>{title}</h3>
+            </div>
+          )}
+
+          {questions.map((question) => {
+            const { id, type } = question;
+            const AnswerComponent = getAnswerComponent(type);
+
+            return <AnswerComponent key={id} {...question} formValues={formValues} />;
+          })}
+        </div>
+        <div className="m-wizard-form__form-footer">
+          {!isFirstPage && (
+            <button className="btn btn-primary" type="button" onClick={onPrevious}>
```

Comment:
All the buttons are not primary. They are transparent with a border and they have an arrow left or right. Could you check the designs?

Reply:
I haven't styled any of the buttons yet (as they don't impact functionality, leaving them for last).
I didn't style the header ones yet. They're also missing the arrows. I styled everything form specific but not buttons.